source/NationalInstruments/jquery.flot.historybuffer.analogWaveform.js

```diff
@@ -119,8 +119,8 @@ console.log(hb1.toDataSeries()); //[[4, 1], [5, 2], [6, 3], [null, null], [1, 1]
 
     function appendWaveformToDecimateBuffer(aw, start, end, buffer) {
         var Y = aw.Y,
```

Comment:
If we're going to fix parameter capitalization, might as well fix `Y` too.

Reply:
Done.